game.rb

```diff
@@ -13,31 +15,42 @@ def initialize(board_dimension, player1_name, player1_sym, player2_name, player2
   def status
 
     rows_status = @board.check_rows
-    return rows_status if rows_status == Board::WIN
-
     columns_status = @board.check_columns
-    return columns_status if columns_status == Board::WIN
-
     main_diagonal_status = @board.check_main_diagonal
-    return main_diagonal_status if main_diagonal_status == Board::WIN
-
     second_diagonal_status = @board.check_second_diagonal
-    return second_diagonal_status if second_diagonal_status == Board::WIN
 
-    draw_status = @board.draw
-    return draw_status if draw_status == Board::DRAW
-    
-    
+    win = [rows_status, columns_status, main_diagonal_status, second_diagonal_status].any? do |status|
+      status == Board::WIN
+    end
+
+    if(win)
+      return Board::WIN
+    else
+      draw_status = @board.draw
+      return draw_status if draw_status == Board::DRAW
+    end
+    Board::ON_GAME
   end
 
   def make_a_move move
-    @player1.play(move)
-    @board.fill_table(@player1.value, @player1.symbol)
-    @player1, @player2 = @player2, @player1
+    @player_on_turn.play(move)
+  end
```

Comment:
How does your scoring work? If I choose cell 9 I get 9 points?

Reply:
Currently, we don't have a scoring system . The cell that is being chosen ,it appears in the board and the check for win status is in process